daily-code/push-item.c

```diff
@@ -0,0 +1,39 @@
+#include<stdio.h>
+#include<stdlib.h>
+#define ms 5
+
+void push();
+int s[5], t=-1;
```

Comment:
give a meaningful name e.g stack[5]

Reply:
what is t? the name should be easy so that reader can understand the code without explanation

Reply:
given meaningful terms to ms and t